src/views/EventViews001.js

```diff
@@ -1,278 +0,0 @@
-/**
```

Comment:
You sure this can be completely removed?  Just asking because there is a lot of code here.